news/2021-06-20-mappers-report-may.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: post
+title: "The Mappers' Report: May"
+date: 2021-06-20 20:00:00 +0000
+---
+
+Take everything in with the May issue of the Mappers' Report and begin your summer up-to-date with the mapping and modding community!
+
+![](/wiki/shared/news/banners/the-mappers-report.jpg)
+
+This month's issue of the Mappers' Report contains the following topics:
+
+- **[Ranking Criteria Changes](#ranking-criteria-changes)**
+- **[Within the Beatmap Nominators and NAT](#within-the-beatmap-nominators-and-nat)**
+- **[Mappers' Showcase](#mappers-showcase)**
+- **[Mapping Contests](#mapping-contests)**
+- **[Around the Community](#around-the-community)**
+- **[Conclusion](#conclusion)**
+
+## Ranking Criteria Changes
+
+Unfortunately, we don't have any Ranking Criteria changes for you this month. But fear not, we have several proposals at hand waiting for further polishing. Be one of the first and check them out right away!
```

Comment:
```suggestion
Unfortunately, we don't have any Ranking Criteria changes for you this month. But fear not, because we have several proposals at hand waiting for further polishing. Be one of the first and check them out right away!
```